package.json

```diff
@@ -13,7 +13,7 @@
     "test": "jest",
     "version": "node ./scripts/version.js",
     "watch": "rollup -c -w",
-    "lint": "eslint --cache --ext .tsx,.ts,.js,.jsx,.md . --fix",
+    "lint": "eslint --cache --ext .tsx,.ts,.js,.jsx,.md .",
```

Comment:
Why did you remove the `--fix`? Is this the part you were telling me about earlier where you said it won't be active yet?

Reply:
Ah, oversight. I removed it temporarily for dev purposes. Adding back.